### PR TITLE
SDIT-2192 Exclude Pilot prisons in incidents reconciliation

### PR DESCRIPTION
**Pull request by karenmillermoj**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/core/ServiceAgencySwitchesService.kt [hunk 1]
```diff
@@ -27,6 +27,11 @@ class ServiceAgencySwitchesService(
     externalServiceRepository.findByIdOrNull(serviceCode)
       ?: throw NotFoundException("Service code $serviceCode does not exist")
 
+  fun getServicePrisonIds(serviceCode: String): List<String> =
```
Comment (karenmillermoj) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/core/ServiceAgencySwitchesService.kt:30]: Realised I've not added a test for this - will do a separate PR

**Review by mikehalmamoj — APPROVED** [review]
